home-data-migration/src/main/resources/db/changelog/0.0.1/Issue33.xml

```diff
@@ -114,10 +85,10 @@
                              primaryKeyName="account_pkey"/>
             </column>
             <column name="CONTACTS" type="VARCHAR">
-                <constraints nullable="false"/>
+                <constraints nullable="true"/>
             </column>
             <column name="CREATE_DATE" type="TIMESTAMP">
-                <constraints nullable="false"/>
+                <constraints nullable="true"/>
```

Comment:
This constraint  must not be  nullable - true

Reply:
Done.